examples/BulmaForm/BulmaForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { FormFieldPropsType } from '../../index';
+import BulmaSelect from './components/BulmaSelect';
+import BulmaInput from './components/BulmaInput';
+import FormBuilder from '../../src/components/FormBuilder/FormBuilder';
+import validate from './validate';
+
+type FieldsType = 'name' | 'city' | 'email';
+
+const BulmaForm = () => {
+  const fields: FieldsType[] = ['name', 'city', 'email'];
+  const inputClass = 'input is-primary';
+  const selectClass = 'select is-primary';
+
+  const selectOptions = {
+    Berlin: 'Berlin',
+    Munich: 'Munich',
+    Belgrad: 'Belgrad',
+    Warsaw: 'Warsaw',
+    Madrid: 'Madrid'
+  };
+
+  const fieldOptions = {
+    name: {
+      label: 'Name:',
+      type: 'text',
+      className: inputClass,
+      component: (props: FormFieldPropsType<any>) => <BulmaInput {...props} />
```

Comment:
That's function is redundant, you can just write :
```suggestion
      component: BulmaInput
```